src/sidebar.ts

```diff
@@ -369,519 +392,524 @@ const guidesSidebar = {
-          dirName: "troubleshooting",
-        },
-        "oauth2-oidc/identity-provider-integration-settings",
-        "guides/upgrade/sdk-v1",
-        "hydra/debug",
-        "hydra/debug/logout",
-        "hydra/debug/token-endpoint-auth-method",
-      ],
-    },
-  ],
+      "oauth2-oidc/identity-provider-integration-settings",
+      "guides/upgrade/sdk-v1",
+      "hydra/debug",
+      "hydra/debug/logout",
+      "hydra/debug/token-endpoint-auth-method",
+    ],
+  },
+]
 
-  migrateToOry: [
-    homeLink,
-    "migrate-to-ory/index",
-    "migrate-to-ory/auth0",
-    "hydra/self-hosted/migrating-from-mitreid",
-  ],
-}
+const migrateToOrySidebar = [
+  homeLink,
+  "migrate-to-ory/index",
+  "migrate-to-ory/auth0",
+  "hydra/self-hosted/migrating-from-mitreid",
+]
 
-const identitiesSidebar = {
-  identities: [
-    homeLink,
-    {
-      type: "category",
-      label: "Concepts",
-      collapsed: false,
-      collapsible: false,
-      link: {
-        type: "doc",
-        id: "identities/index",
+const identitiesSidebar = [
+  homeLink,
+  {
+    type: "category",
+    label: "Concepts",
+    collapsed: false,
+    collapsible: false,
+    link: {
+      type: "doc",
+      id: "identities/index",
+    },
+    items: [
+      "security-model",
+      "identities/native-browser",
+      "concepts/redirects",
+    ],
+  },
+  {
+    type: "category",
+    label: "Self-service user flows",
+    collapsed: false,
+    collapsible: false,
+    link: {
+      type: "doc",
+      id: "kratos/self-service",
+    },
+    items: [
+      "kratos/concepts/browser-redirect-flow-completion",
+      "kratos/self-service/flows/user-registration",
+      "kratos/self-service/flows/user-login",
+      "kratos/self-service/flows/user-logout",
+      "kratos/self-service/flows/user-settings",
+      "kratos/self-service/flows/verify-email-account-activation",
+      "kratos/self-service/flows/account-recovery-password-reset",
+      "kratos/self-service/flows/user-facing-errors",
+    ],
+  },
+  {
+    type: "category",
+    label: "Authentication",
+    collapsed: false,
+    collapsible: false,
+    link: {
+      type: "doc",
+      id: "kratos/concepts/credentials",
+    },
+    items: [
+      "concepts/password-policy",
+      "kratos/passwordless/passwordless",
+      "kratos/mfa/overview",
+    ],
+  },
+  {
+    type: "category",
+    label: "User interface",
+    collapsed: false,
+    collapsible: false,
+    link: {
+      type: "doc",
+      id: "kratos/bring-your-own-ui/custom-ui-overview",
+    },
+    items: [
+      "account-experience/index",
+      "kratos/bring-your-own-ui/configure-ory-to-use-your-ui",
+    ],
+  },
+  {
+    type: "category",
+    label: "Send Emails & SMS",
+    collapsed: false,
+    collapsible: false,
+    link: {
+      type: "doc",
+      id: "kratos/emails-sms/custom-email-templates",
+    },
+    items: ["kratos/emails-sms/custom-email-templates"],
+  },
+]
+
+const oauth2Sidebar = [
```

Comment:
I was not able due to time constraints to do the flattening option for all nav items, maybe you could follow up on this @vinckr ?

Reply:
so identitiesSidebar, oauth2Sidebar, permissionsSidebar, and selfhostingSidebar need flattening still? 

should I do it in this PR or do you want to merge asap?

Reply:
I think it's fine in another PR, this already improves the state I think

Reply:
basically all sidebars that don't have a function call yet